index.bs

```diff
@@ -38,16 +38,50 @@ urlPrefix: https://www.w3.org/TR/html5/
-Other interfaces, such as device orientation events, can be repurposed to surface VR input but doing so dilutes the interface's original
-intent and often does not provide the precision necessary for high-quality VR. The WebVR API provides purpose-built interfaces
-to VR hardware to allow developers to build compelling, comfortable VR experiences.
+Other interfaces, such as device orientation events, can be repurposed to surface VR/MR input but doing so dilutes the interface's original
+intent and often does not provide the precision necessary for high-quality VR/MR. The WebVR API provides purpose-built interfaces
+to VR/MR hardware to allow developers to build compelling, comfortable VR/MR experiences.
+
+## Terminology ## {#intro-terminology}
+For the purposes of this specification, the following VR/MR specific terms are defined.
+
+### Content locking types ### {#intro-terminology-contentlockingtypes}
+
+<dfn>World-locked</dfn>
+Content that has a stationary position in the user's environment regardless of where the user is standing or which direction the user is looking.
+
+<dfn>Body-locked</dfn>
+Content that has a stationary orientation relative to the user's environment.  If the user changes orientation, the content stays put.  If the user changes position, the content tags along.
+
+<dfn>Face-locked</dfn>
+Content that is not related to the user's environment.  Regardless of the user changing orientation or position, the content stays at the same place in the user's field of view.
+
+<dfn>Head-locked</dfn>
+The same meaning as face-locked.
+
+
+### Experience Types ### {#intro-terminology-experiencetypes}
+
+<dfn>3DOF experience</dfn>
+An experience that does not require or allow a user to change their physical position in space.  These experiences are based on the user's head orientation and may include some amount of neck-modeling.  An example of this category of experience is a 360 degree video.
+
+<dfn>Seated experience</dfn>
+An experience that relies on knowledge of a user's position in space but does not rely on a floor plane.  An example of this category of experience is a racing game that allows a user to lean into turns.
+
+<dfn>Standing experience</dfn>
+An experience that relies on knowledge of the floor plane but does not encourage users to walk around.  An example of this category of experience is game of cricket or baseball.
+
+<dfn>Room-scale experience</dfn>
+An experience that utilizes knowledge of the floor plane and encourages users to walk around within specific bounds.  An example of this category of experience is CAD modeling which allows a user to walk around the object being modeled.
+
+<dfn>World-scale experience</dfn>
+An experience that takes advantage of the ability to walk anywhere without bounds.  In such experiences, there is no single floor plane.  An example of this category of experience is turn-by-turn directions within a multistory building.
```

Comment:
I think you may want to differentiate between "buidling-scale" and "world-scale".  As we move beyond "a single room with a single ground plane" into "a local world that is more complex" there are issues to be dealt with.  But, a local 3D coordinate system is still sufficient to relate things to each other and do rendering.  As you move to geospatial coordinates (true world-scale), the use of a local coordinate system becomes difficult.  For example, a natural coordinate system for the world is ECEF (earth-centered earth-fixed), which has the "ground plane from the user's viewpoint" not sitting on the X/Z plane with Y up (rather the ground plane is a tangent plane to the earth).   So, perhaps world-scale isn't the right term here.